yarn.lock

```diff
@@ -5866,7 +5866,25 @@ node-pre-gyp@^0.6.29, node-pre-gyp@~0.6.32:
     tar "^2.2.1"
     tar-pack "^3.4.0"
 
-node-sass@^3.10.1, node-sass@^3.4:
+node-sass@3.4.2:
```

Comment:
The newest version of node-sass is 4.5.3. That version should get installed if you run `yarn upgrade node-sass`.